samples/XCT.Sample.Android/Xamarin.CommunityToolkit.Sample.Android.csproj

```diff
@@ -127,7 +127,7 @@
   <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.CSharp.targets" />
   <ProjectExtensions>
     <VisualStudio>
-      <UserProperties XamarinHotReloadXFormsNugetUpgradeInfoBarXamarinCommunityToolkitSampleAndroidHideInfoBar="True" />
+      <UserProperties TriggeredFromHotReload="False" XamarinHotReloadXFormsNugetUpgradeInfoBarXamarinCommunityToolkitSampleAndroidHideInfoBar="True" />
```

Comment:
please revert this change

Reply:
Reverted. It's added automatically every time I launch Android sample. Is there some reason it cannot be checked into source control?

Reply:
FYI: https://developercommunity.visualstudio.com/content/problem/1291161/xamarinandroid-csproj-is-always-shown-as-edited-in.html